include/mitsuba/render/blender_types.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
```

Comment:
I don't think these need to be a top-level public API (which is what putting a header file into ``include/mitsuba`` means). I propose adding this to the ``src`` directory similar to what was done in Mitsuba 1.